docs/5.x/installation.md

```diff
@@ -216,15 +207,43 @@ Tue Apr 10 14:01:09 UTC	Executing install phase "/" finished in 5 minutes
 
 ```
 
-The following CLI flags are useful to manage the install operation:
+To abort an installation, press Ctrl+C two times in a row in the terminal.
+If the operation is aborted, the partial cluster state will be automatically removed.
 
+Aborting an agent (and not the installer process), will only prevent this node from joining.
```

Comment:
I think this needs to be clearer. Maybe instead of saying aborting an agent, aborting a join? Then it's linked to what the user is doing, and not the internals of the way we see the implementation?